07-routing-gossip.md

```diff
@@ -300,6 +310,11 @@ The following table specifies the meaning of the individual bits:
 The creating node MUST set `signature` to the signature of the
 double-SHA256 of the entire remaining packet after `signature` using its own `node_id`.
 
+The creating node MUST set `chain_hash` to match 32-byte hash that
+uniquely identifies the channel within the `channel_announcement` message.
+For the Bitcoin blockchain, the `chain_hash` value MUST be (encoded in
+hex): `000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f`.
```

Comment:
Good point. Where does everyone think this definition should go? In the introduction, perhaps? 

I ended up sticking it in the glossary, but there may be a better spot for it.